src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate

```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
   <TemplateData>
 	<Name>Cross-Platform App (Uno Platform)</Name>
-	<Description>Solution template for creating a cross-platform XAML app with the Uno Platform that targets UWP, Android, iOS, and WebAssembly</Description>
+	<Description>Template for creating a multi-platform C# & XAML application targeting WebAssembly, Android, iOS, macOS and Linux</Description>
```

Comment:
Can we add "Solution" in front of "Template" please. I am also using this to practice gitthub ocllavoration - never approved anything before :)

Reply:
Also - if we can use Multi-platform in place of Cross platform that is better.